### ISSUE TITLE
login loop - cannot login to Company Communicator

### ISSUE THREAD

**Issue by Craigster19** [OfficeDev/microsoft-teams-apps-company-communicator#343]
I have created a package, followed the instructions, double and triple checked my TenantID settings and every time Teams tries to log into the app I get into a login loop. Saw all the other issues user have had doing their installs. So, I uninstalled the package, created a second package, double and triple checked all the settings and I am getting the same results. Any ideas!

**Comment by ChrisStoneham** [comment]
@Craigster19 I'm getting the same issue - don't suppose you've had any joy resolving this?

**Comment by Craigster19** [comment]
I have completely uninstalled everything and will be doing a re-install with Microsoft on the call, next week, watching to make sure everything is correct. Will know more than

**Comment by Zipper08** [comment]
@Craigster19 Also interested if you managed to resolve this issue, just setup everything, having the same issue

**Comment by Hildagb27** [comment]
Having the same issue, it keep asking "please sign in to continue. "

**Comment by Craigster19** [comment]
Sorry for the delay in updating my question. YES, I got it working. I worked with MS on the call watching and the one thing I did different this 3rd time was I did not use the words "Company" and/or "Communicator" when creating the applications, the Resource Group and Base Name Resource, I used something completely different. The only time I used the words Company Communicator was when creating the zip files. Not completely sure if this was the fix, but it works now.
